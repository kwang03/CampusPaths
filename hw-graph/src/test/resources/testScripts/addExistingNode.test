#Tests behavior of adding a node that already exists in the graph

#Create graph
CreateGraph myGraph

#Adds nodes and then lists the nodes in the graph
AddNode myGraph n1
ListNodes myGraph

AddNode myGraph n1
ListNodes myGraph