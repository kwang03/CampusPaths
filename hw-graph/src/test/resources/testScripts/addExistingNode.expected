#Tests behavior of adding a node that already exists in the graph

#Create graph
created graph myGraph

#Adds nodes and then lists the nodes in the graph
added node n1 to myGraph
myGraph contains: n1

added node n1 to myGraph
myGraph contains: n1